wcivf/apps/people/models.py

```diff
@@ -51,6 +51,15 @@ def get_local_party(self):
         else:
             return None
 
+    @property
+    def is_welsh_run(self):
+        """
+        Return a boolean if this is a Welsh government run ballot
+        """
+
+        if self.post.nice_territory() == "Wales":
```

Comment:
Since (I don't think) we don't have access to `NUTS` keys in WCIVF,  I wasn't sure the best way to find the country. I am confused as to why 'local' seems to fall outside of country definitions rather than within.

Reply:
I dont think we actually need this, as we can instead just check if the `previous_party_affiliations` exist. The reason for the `is_welsh_run` helper in YNR is about stopped the relations being added on ballots that shouldn't be possible. But in WCIVF there is no update/creating happening, so it is not so important

Reply:
Sorry, this was my suggestion because I wanted to prevent unneeded queries! As long as we're using `select_related` on them it should be ok, but I'd like to ensure that

Reply:
I have added the `prefetch_related` on the method that [gets people for the postcode view](https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1120/files#diff-0e232cd028b7a1c7c67eec353c08f2a2aa146ca5359037faf9fa1fb06da109b6R116) but we will need something similar on the person view also.

Reply:
@VirginiaDooley in the `get_person` method you may want to try adding:
```
        queryset = queryset.prefetch_related(
            "personpost__previous_party_affiliations",
        )
```
around line 32, before the Try/Except block